docs/snippets/deploy-func-intro.md

```diff
@@ -0,0 +1,4 @@
+<!-- Snippet used in the following topics:
+- /docs/getting-started/build-run-deploy-func.md
+-->
+Deploying a function creates an OCI container image for your function, and pushes this container image to your cluster image registry. The function is deployed to the cluster as a live Knative Service. Redeploying a function updates the container image and resulting live Service that is running on your cluster. Functions that have been deployed to a cluster are accessible on the cluster just like any other Knative Service.
```

Comment:
```suggestion
Deploying a function creates an OCI container image for your function, and pushes this container image to your image registry. The function is deployed to the cluster as a Knative Service. Redeploying a function updates the container image and resulting Service that is running on your cluster. Functions that have been deployed to a cluster are accessible on the cluster just like any other Knative Service.
```
A couple of nits. Deploying does push to a container registry, but it does not have to be a registry on the cluster. I think leaving out the word "cluster" here is clearer. Regarding "live Service", I can infer what that means, but I have not seen it used anywhere else and wonder what a non-live Service is.